task_5.py

```diff
@@ -1,6 +1,16 @@
 def factorial(n):
-    pass
-
+    silnia = 1
+    for i in range(n,n +1):
+        if n==0 or n==1:
+            return 1
+        else:
+            for i in range (2, n+1):
+                silnia = silnia*i
+            return silnia           
 
 def report():
-    pass
\ No newline at end of file
+    for j in range(101):
+        silnia = factorial(j)
+        dlugosc = len(str(silnia))
```

Comment:
Staraj się nie używać polskich słow zmieszanych z angielskimi, bo to się źle potem czyta.